README.md

```diff
@@ -42,21 +42,13 @@ To plot the output as a heatmap:
 
 # Develop
 
-Continually build and test while developing. This will automatically create your virtual env
+The C code (`pyFlowSOM/flowsom.c`) is wrapped using Cython (`pyFlowSOM/cyFlowSOM.cyx`).
 
-    ./build.sh && ./test.sh
+Tests do an approximate exact comparison to cluster id groundtruth and an approximate comparison to node values only because of floating point differences. Randomness works in tandem to the `seed` flag to the `som` function.
```

Comment:
I'm also confused by this description. What is "approximate exact comparison"?

Reply:
Forgot to remove `approximate` from this portion.